@Doyuni 
HI, 
coveralls was pending with both jdk8 and jdk11,
i have no idea, i will try this..

### jdk8 log
- https://travis-ci.org/github/ztkmkoo/dss/jobs/714473116
-  No coverage report files found
```
[INFO] ------------------------------------------------------------------------

[INFO] Reactor Summary for DSS PARENT 0.4.3:

[INFO] 

[INFO] DSS PARENT ......................................... FAILURE [  2.690 s]

[INFO] DSS COMMON ......................................... SUCCESS [  0.766 s]

[INFO] DSS CORE ........................................... SUCCESS [  8.412 s]

[INFO] DSS ................................................ SUCCESS [  4.689 s]

[INFO] ------------------------------------------------------------------------

[INFO] BUILD FAILURE

[INFO] ------------------------------------------------------------------------

[INFO] Total time:  17.855 s

[INFO] Finished at: 2020-08-03T13:46:01Z

[INFO] ------------------------------------------------------------------------

[ERROR] Failed to execute goal org.eluder.coveralls:coveralls-maven-plugin:4.3.0:report (default-cli) on project dss: I/O operation failed: No coverage report files found -> [Help 1]

[ERROR] 

[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.

[ERROR] Re-run Maven using the -X switch to enable full debug logging.

[ERROR] 

[ERROR] For more information about the errors and possible solutions, please read the following articles:

[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

### jdk 11
- https://travis-ci.org/github/ztkmkoo/dss/jobs/714473117
- Skip property set, skipping plugin execution
``` 
[INFO] Skip property set, skipping plugin execution

[INFO] ------------------------------------------------------------------------

[INFO] Reactor Summary for DSS PARENT 0.4.3:

[INFO] 

[INFO] DSS PARENT ......................................... SUCCESS [  5.039 s]

[INFO] DSS COMMON ......................................... SUCCESS [  1.050 s]

[INFO] DSS CORE ........................................... SUCCESS [ 11.894 s]

[INFO] DSS ................................................ SUCCESS [  5.503 s]

[INFO] ------------------------------------------------------------------------

[INFO] BUILD SUCCESS

[INFO] ------------------------------------------------------------------------

[INFO] Total time:  25.149 s

[INFO] Finished at: 2020-08-03T14:03:20Z

[INFO] ------------------------------------------------------------------------
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,7 @@
 language: java
 jdk:
   - openjdk8
+  - openjdk11
 install: true
 addons:
   sonarcloud:

--- a/pom.xml
+++ b/pom.xml
@@ -15,6 +15,42 @@
         <module>dss-example</module>
     </modules>
 
+    <profiles>
+        <profile>
+            <id>jdk11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <properties>
+                <java.version>11</java.version>
+                <access.option>--illegal-access=permit</access.option>
+            </properties>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>org.eluder.coveralls</groupId>
+                            <artifactId>coveralls-maven-plugin</artifactId>
+                            <configuration>
+                                <skip>true</skip>
+                            </configuration>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+            </build>
+        </profile>
+        <profile>
+            <id>jdk8</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <jdk>1.8</jdk>
+            </activation>
+            <properties>
+                <java.version>1.8</java.version>
+            </properties>
+        </profile>
+    </profiles>
+
     <name>DSS PARENT</name>
     <description>Distributed server system</description>
     <url>https://github.com/ztkmkoo/dss</url>
@@ -92,7 +128,14 @@
             <version>${lombok.version}</version>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.2.3</version>
+        </dependency>
     </dependencies>
+
 
     <build>
         <plugins>

--- a/pom.xml
+++ b/pom.xml
@@ -79,9 +79,8 @@
 
     <properties>
         <!-- Generic properties -->
-        <java.version>1.8</java.version>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <!-- Dependencies version -->

--- a/pom.xml
+++ b/pom.xml
@@ -15,42 +15,6 @@
         <module>dss-example</module>
     </modules>
 
-    <profiles>
-        <profile>
-            <id>jdk11</id>
-            <activation>
-                <jdk>11</jdk>
-            </activation>
-            <properties>
-                <java.version>11</java.version>
-                <access.option>--illegal-access=permit</access.option>
-            </properties>
-            <build>
-                <pluginManagement>
-                    <plugins>
-                        <plugin>
-                            <groupId>org.eluder.coveralls</groupId>
-                            <artifactId>coveralls-maven-plugin</artifactId>
-                            <configuration>
-                                <skip>true</skip>
-                            </configuration>
-                        </plugin>
-                    </plugins>
-                </pluginManagement>
-            </build>
-        </profile>
-        <profile>
-            <id>jdk8</id>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-                <jdk>1.8</jdk>
-            </activation>
-            <properties>
-                <java.version>1.8</java.version>
-            </properties>
-        </profile>
-    </profiles>
-
     <name>DSS PARENT</name>
     <description>Distributed server system</description>
     <url>https://github.com/ztkmkoo/dss</url>
@@ -91,6 +55,28 @@
         </repository>
     </distributionManagement>
 
+    <profiles>
+        <profile>
+            <id>jdk11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <properties>
+                <java.version>11</java.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>jdk8</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <jdk>1.8</jdk>
+            </activation>
+            <properties>
+                <java.version>1.8</java.version>
+            </properties>
+        </profile>
+    </profiles>
+
     <properties>
         <!-- Generic properties -->
         <java.version>1.8</java.version>
@@ -103,7 +89,7 @@
         <awaitility.version>4.0.3</awaitility.version>
         <lombok.version>1.18.12</lombok.version>
         <coveralls.version>4.3.0</coveralls.version>
-        <jacoco.version>0.7.6.201602180812</jacoco.version>
+        <jacoco.version>0.8.3</jacoco.version>
         <surefire.version>2.22.2</surefire.version>
     </properties>
 
@@ -128,14 +114,7 @@
             <version>${lombok.version}</version>
             <scope>provided</scope>
         </dependency>
-
-        <dependency>
-            <groupId>javax.xml.bind</groupId>
-            <artifactId>jaxb-api</artifactId>
-            <version>2.2.3</version>
-        </dependency>
     </dependencies>
-
 
     <build>
         <plugins>
@@ -143,6 +122,13 @@
                 <groupId>org.eluder.coveralls</groupId>
                 <artifactId>coveralls-maven-plugin</artifactId>
                 <version>${coveralls.version}</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>javax.xml.bind</groupId>
+                        <artifactId>jaxb-api</artifactId>
+                        <version>2.3.1</version>
+                    </dependency>
+                </dependencies>
             </plugin>
 
             <plugin>

